.github/workflows/tests.yml

```diff
@@ -0,0 +1,56 @@
+name: Tests
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+  workflow_dispatch:
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  tests:
+    timeout-minutes: 45
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        os: [ubuntu-20.04]
+        pyv: ["3.8"]
+        reqs: ["earliest", "latest", "dev"]
```

Comment:
Just trying to get feature parity with the old azure-pl-based workflow. Obviously we can test more python versions and more platforms. And also testing 3 reqs is kinda questionable too.